client/src/utils/Validation.ts

```diff
@@ -181,6 +182,23 @@ export const checkDimension = (param: {
   return checkMultiple({ value, multipleNumber });
 };
 
+/**
+ * function to check whether value(type: number) is equal to specified value or in valid range
+ * @param param specValue and params checkRange function needed
+ * @returns whether input is valid
+ */
+export const checkSpecValueOrRange = (param: {
+  value: number;
+  min: number;
+  max: number;
+  compareValue: number;
+}): boolean => {
```

Comment:
If i only need to check value === compareValue, do i need to pass min and max?

Reply:
we already have a method to check comparison, `specValueOrRange` is for condition like `{-1} ∪ [10, 20]`, so you should pass min or max if you use it